### PR TITLE
Recipe definition added 

### PR DESCRIPTION
**Pull request by Aaishpra**
Recipe definition added to glossary and links in homepage added

PR Checklist:

image

Fixes: #1389 

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/misc/00_intro.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Glossary
 
 .. glossary::
 
+  Recipe
+    **R**\ ecipe is the collection of files required to build the conda package. This includes at minimum the `meta.yaml <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml>`_, but can also include license files, patches, build scripts, test scripts etc.
```
Comment (ForgottenProgramme) [inline src/misc/00_intro.rst:10]: Recipe is the collection of files required to build a (not the) conda package. The includes (comma) at minimum (comma) the meta.yaml file...
Maybe you can add a link to the sample recipe. (https://github.com/conda-forge/staged-recipes/tree/master/recipes/example)

Comment (Aaishpra) [reply src/misc/00_intro.rst]: Done, thanks for suggesting

**Review by viniciusdc — COMMENTED** [review]
I think this addition to the glossary is valid, but the link in the main page still looks kinda strange to me (though I liked it )

**Comment by Aaishpra** [comment]
Basically we don't have separate links for every definition in the glossary, ( although i think there should be) so it felt strange to me also. But if you want we can remove the links from homepage.

**Comment by viniciusdc** [comment]
> Basically we don't have separate links for every definition in the glossary, ( although i think there should be) so it felt strange to me also. But if you want we can remove the links from homepage.

@beckermr any thoughts on this? (My idea was set it and check if someone complains ;p)

**Comment by Aaishpra** [comment]
@viniciusdc @beckermr  now as I think more, I think there is absolutely a need for separate links/references to different glossary descriptions so that they can be properly referenced in other places.
Something like [here](https://conda-forge.org/docs/maintainer/conda_forge_yml.html#azure), where we have different sub-headings as well as references for every config.
If you think this can be worked on, I would like to open an issue and work on the same.

**Comment by beckermr** [comment]
I don't have a preference here.

**Comment by jaimergp** [comment]
Integrated in #2087. Thanks!

**Pull request closed without merging**